pkg/network/usm/ebpf_ssl.go

```diff
@@ -490,25 +485,28 @@ func (o *sslProgram) Name() string {
 	return "openssl"
 }
 
-// ConfigureOptions changes map attributes to the given options.
-func (o *sslProgram) ConfigureOptions(options *manager.Options) {
+func sharedLibrariesConfigureOptions(options *manager.Options, cfg *config.Config) {
```

Comment:
in the config we are using only the value of the MaxTrackedConnections, should we consider passing to the function only the value?

Reply:
I'll do that in a different PR after all modules are independendent, as it is repeated in the three modules